src/index.ts

```diff
@@ -369,9 +369,9 @@ const transform = async ({
   const middlewareEntries = Object.values(middlewareManifest.middleware);
   const functionsEntries = Object.values(middlewareManifest.functions);
   for (const [name, filepath] of functionsMap) {
-    if (name === "middleware" && middlewareEntries.length > 0) {
+    if (middlewareEntries.length > 0 && (name === "middleware" || name === "src/middleware")) {
       for (const entry of middlewareEntries) {
-        if ("middleware" === entry?.name) {
+        if (entry?.name === "middleware" || entry?.name === "src/middleware") {
```

Comment:
this feels slightly clunky, could you maybe create a utility function? 🙏 

for examples:
```ts
function isMiddlewareDir(dirName: string): boolean {
  return /^(?:src\/)?middleware$/.test(dirName);
}
```

so that then the code can be updated to something like:
```ts
    if (middlewareEntries.length > 0 && isMiddlewareDir(name)) {
      for (const entry of middlewareEntries) {
        if (entry?.name && isMiddlewareDir(entry.name)) {
```

what do you think? 🙂

Reply:
sure, a utility function would make sense. i just didn't wanted to introduce bigger changes initially, hence i choose to simply add it in place..

Reply:
ah I see, thanks for trying to reduce the change 🙂, I'm pretty sure different people will have different opinions on this (especially since it's a small change) but yeah to me a utility function here helps in making the code (slightly) more dry and more readable, so definitely a plus from my point of view 😃 

(I'm am planning to also slit the index.ts file in multiple files at some point so adding utilities should be nicer in the future since there would be more structure to the code)

Reply:
i would do a refactoring separated from this pr. when you start splitting index.ts into smaller chunks anyway you would have a better overview on which functions needs to be extracted and where.